test/REQUIRE

```diff
@@ -1,3 +1,2 @@
-Lint
```

Comment:
You can just specify `Lint 0.1.0 0.2.0` to avoid installing the most recent release of Lint.jl.